projects/sbb-esta/angular-public/file-selector/src/_file-selector.scss

```diff
@@ -11,18 +11,28 @@
 
 @mixin fileSelectorButton() {
   @include buttons;
+  @include secondaryButton;
 
-  &:not(.sbb-file-selector-disabled) {
-    @include secondaryButton;
-    @include arrowbuttons;
+  @if $sbbBusiness {
+    .sbb-svgsprite-icon {
+      display: none;
+    }
 
+    &.sbb-file-selector-disabled {
+      opacity: 50%;
```

Comment:
I'd not use opacity because it depends on the background which can be non white

Reply:
I took that styling from the InVision data. In the future I'll ignore opacity where it doesn't make sense and use colors instead

Reply:
Very interesting, I couldn't find it there, where exactly did you find that?

Reply:
Ah well it was actually just the text that was marked that way. I must've interpreted it wrong. Nonetheless I now also made the text not opaque but the right colour.
![image](https://user-images.githubusercontent.com/17550954/77994798-eec61100-732a-11ea-8d5f-641cc8b7f1f7.png)

Reply:
confusing, but thank you for your screenshot!

I think the background color is still not the right one. Compare Screenshot:
invision:
![image](https://user-images.githubusercontent.com/29278232/78020974-aa4e6b80-7352-11ea-9e0a-078664419103.png)

https://angular-staging.app.sbb.ch/352/business/components/file-selector
![image](https://user-images.githubusercontent.com/29278232/78021039-c6520d00-7352-11ea-8028-6171d54375a5.png)

Reply:
my color picker sais #f1f1f1 which is not defined in sbb color palete... nearest color I think is $sbbColorCloud. @Frenggi Maybe you could solve our confusion on the right font and background color of the disabled file selector button in business?

Reply:
Hi @jeripeierSBB, @vlanz 
The method with opacity we chose for all variants of the button. Because we could not find colours in the SBB colour palette for disabled buttons (especially for the primary button), we decided to take the opacity way.
Because the complete design is defined for white background and we tried to support accessibility (contrasts) as far as possible, this should work that way.

--> background color: same as the enabled button
--> background opacity: 40%
--> text color: same as the enabled button
--> text opacity: 50%